src/@next/Switch/SwitchStyle.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components';
+import { Breakpoints, Spacing } from '..';
+
+import { Blue, Neutral } from '../utilities/colors';
+import { SwitchProps } from './Switch';
+
+export const SwitchStyle = styled.div<Pick<SwitchProps, 'disabled'>>`
+  position: relative;
+  width: ${Spacing.space40};
+  height: ${Spacing.space24};
+
+  ${({ disabled }: Pick<SwitchProps, 'disabled'>) => ({
```

Comment:
Since `disabled` only hold `true` or `false`, use `data-disabled` instead. That's the convention on this project.

Reply:
if i put the prop like this for the InputStyle

```
 data-disabled={disabled}
```

the switch can still be toggled

<img width="296" alt="image" src="https://user-images.githubusercontent.com/4475526/219995215-46454ce2-2ac2-4fc5-9959-a48e9d8dbf65.png">

Reply:
Hmmm, what I intended was to prevent Passing Props to styled styles. Instead, use `[data-disabled=true]`

Reply:
https://github.com/glints-dev/glints-aries/pull/736#discussion_r1091630075

Reply:
should be fixed, you can check agian